src/components/Settings/Settings.css

```diff
@@ -23,7 +23,7 @@
 }
 
 .Settings__section--secondaryAction {
-  padding-right: 1.5rem;
+  padding-right: 0rem;
```

Comment:
We can use `0` instead of `0rem`

Reply:
Also i think this change is not related to the people component? it's necessary?